vendor.conf

```diff
@@ -73,7 +73,7 @@ github.com/emicklei/go-restful ff4f55a206334ef123e4f79bbf348980da81ca46
 github.com/emicklei/go-restful-swagger12 1.0.1
 github.com/pkg/errors v0.8.0
 github.com/godbus/dbus v4.0.0
-github.com/urfave/cli v1.19.1
+github.com/urfave/cli v1.20.0
```

Comment:
The one lingering question that I had was the number/value to use here. 1.20.0 is the latest stable release and I think that's the right one for us (vs 2.0 for beta).  However Buildah uses "master" here which translates to the latest stable release.  I think we ought to be consistent in CRI-O, Buildah and Skopeo.  Should they all be 1.20.0 or "master"?

Reply:
no point in being consistent, they are different projects and one can use whatever they want in their project.